sources/packages/api/src/services/student/student.service.ts

```diff
@@ -49,6 +50,12 @@ export class StudentService extends RecordDataModelService<Student> {
       phone: otherInfo.phone,
     };
     student.user = user;
+
+    // Get PD status from Archive DB
+    const result: { permanent_disability_flg: string | null } = await this.archiveDB.getIndividualPDStatus(student);
```

Comment:
I think that it is fine for now, but in the future, I would prefer to create interfaces for the returning objects from the ArchiveDB methods.

Reply:
I will create interface.